### PR TITLE
Use PackedForward instead of Message

### PR DESCRIPTION
**Pull request by okkez**
See https://github.com/fluent/fluentd/issues/671

### CODE REVIEW

File: lib/fluent/logger/fluent_logger.rb [hunk 1]
```diff
@@ -153,23 +162,25 @@ def write(msg)
           if @pending
             @pending << data
           else
-            @pending = data
+            @pending = [data]
```
Comment (repeatedly) [inline lib/fluent/logger/fluent_logger.rb]: Sorry. My idea is `@pending` should be Hash, `{'tag1' => "msgpack stream", 'tag2' => "msgpack stream", ...}`.
Using this structure, we can send multiple events in one TCP write.

File: lib/fluent/logger/fluent_logger.rb [hunk 2]
```diff
@@ -55,6 +57,9 @@ def initialize(tag_prefix = nil, *args)
         @host = options[:host]
         @port = options[:port]
 
+        @require_ack_response = options[:require_ack_response]
```
Comment (repeatedly) [inline lib/fluent/logger/fluent_logger.rb]: Could you separate PR?
`require_ack_response` is not a main of `PackedForward` message.

Comment (okkez) [inline lib/fluent/logger/fluent_logger.rb]: OK. I will separate PR.

File: lib/fluent/logger/fluent_logger.rb (hunk 1 above)
Comment (okkez) [inline lib/fluent/logger/fluent_logger.rb]: I got it. I will fix this.

**Pull request closed without merging**